scripts/vidarr-build

```diff
@@ -202,5 +203,9 @@ if args.command == "deploy":
                 ok = False
             else:
                 print(f"Registered on {vidarr_url} as {name}/{args.version}")
+                registered = True
+    if not registered:
+        print("Could not find a server that wanted this workflow. :-(")
```

Comment:
Is there a fix or SOP we can suggest to the user?

Reply:
Okay. Gave them some advice.